### ISSUE TITLE
Mismatched Columns Error with SLEAP Tracking File

### ISSUE THREAD

**Issue by autumnca** [sgoldenlab/simba#188]
I'm attempting to import tracking data from SLEAP of one mouse with four key points. I have looked at a few other issue discussion boards on this GitHub, but they were not too helpful. The file I'm trying to import has tracked or predicted points throughout most of the video, and there are many frames where certain key points are occluded. When I try to import the .slp file I'm met with a error stating "cannot set a row with mismatched columns" as shown by the picture below. 

![image](https://user-images.githubusercontent.com/107413137/174305640-819831cb-260d-41b5-bc79-0f245f4f2ea2.png)

I also tried converting the tracking data from SLEAP to an h5 file and uploading that instead, to which I am met with simba telling me "other, non-skeleton files" are found in the folder I linked it to. It gives me the same response for all other tracking types, despite this file using a skeleton-based tracking method. 

![image](https://user-images.githubusercontent.com/107413137/174307095-10a0b49f-f1fb-4387-9120-ee2472a45914.png)

Let me know what you think some good next steps might be, if you have any questions, or if there is another discussion board I should check out. Thanks so much!

**Comment by autumnca** [comment]
I definitely can do that! I don't have too much to share at the moment except for a couple .slp files (I'm mostly just testing things out right now) but if that would still help out then sure :)

And when I go to the project_bp file, there is a row for each body part. I noticed that the way I labeled them in SLEAP is different from how they are titled there, so I did change what was in that file to match what is in SLEAP. After doing that, I still have the same error. Let me know if I should change that back too. 

I don't have an error or any information show in the terminal when I try to upload my .slp file, but I do when I try to upload an h5 file.

**Comment by sronilsson** [comment]
Yes, one or two slp files with predictions, their associated video files, and the simba project is ideal! 

The SimBA names vs. the sleap names should not matter, and the error lies somewhere else, it might be related to the animal numbers or tracks present in the sleap file. However I will take a look at it and solve it as I will go through the code anyways. You can share the files through a link here or to sronilsson@gmail.com
